doc/ref_arch/kubernetes/chapters/chapter08.md

```diff
@@ -16,10 +16,66 @@ While this Reference Architecture is being developed, Gaps will be identified th
 <a name="8.2"></a>
 ## 8.2 Gap analysis
 
+- Container Run-Time Interfaces towards NFVI resources.
+- Multi-Tenancy
+- K8s as VM based VNF Orchestrator
+
+<a name="8.2.1"></a>
+### 8.2.1 Container run-time Interfaces towards NFVI resources
+
+> This is the southbound infrastructure resources from the container platform as presented by the IaaS provider.
+
+> e.g. network interface type that is presented to a running container.
+
+
+<a name="8.2.2"></a>
+### 8.2.2 Multi-tenancy within Kubernetes
+
+> Today, Kubernetes lacks hard multi-tenancy capabilities<sup>citations</sup>
+
+> Ability to allow untrusted tenants to share infrastructure resources.
+
+
+<a name="8.2.3"></a>
+### 8.2.3 Kubernetes as a VM-based VNF Orchestrator
+
+> In order to support a transition from VNFs only to VNFs and CNFs in the same environment.
+
+
+<a name="8.2.4"></a>
+### 8.2.4 Multiple network interfaces on Pods
+
+> As well as having multiple network interfaces on Pods (e.g. Multus), being able to have different network interfaces in differnet Pods using different CNI plugins within the same cluster.
```

Comment:
We should also have the same features available for all network interfaces.

Reply:
agree